app/main.py

```diff
@@ -1 +1,14 @@
-# write your code here
+from os import remove, mkdir
+
+
+def move_file(command: str):
+    command, file1, file2 = command.split()
+    if command != "mv":
+        return "Incorrect command"
+    if len(file2.split("/")) > 1:
+        for i in range(len(file2.split("/") - 1)):
+            mkdir(file2.split("/")[i])
+    with open(file1, "r") as f1, open(file2, "w") as f2:
+        f2.write(f1.read())
```

Comment:
Do you see these comments?

Reply:
I do not understand what is wrong here, I made a condition that checks the command

Reply:
f1 and f2 bad names for a variables)